awx/main/apps.py

```diff
@@ -1,9 +1,10 @@
 from django.apps import AppConfig
 from django.utils.translation import gettext_lazy as _
+
+from awx.conf import fields, register
```

Comment:
let's not include changes to this module in the PR to make it smaller

Reply:
I have resolved this via this commit: https://github.com/ansible/awx/pull/15537/commits/3e66bc8ce2db82e8b6f4d4811dbdbabd64494abf I'm going to go ahead and resolve this conversation to keep the comments tidy.